### PR TITLE
Typescript support

### PR DESCRIPTION
**Pull request by lukasbuenger**

### CODE REVIEW

File: eslint/eslint-typescript.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  ],
+  plugins: ['@typescript-eslint'],
+  rules: {
+    '@typescript-eslint/explicit-module-boundary-types': 'off',
+  },
+  overrides: [
+    {
+      files: ['*.js', '*.jsx'],
+      rules: {
+        '@typescript-eslint/no-var-requires': 'off',
```
Comment (GianlucaGuarini) [inline eslint/eslint-typescript.js:15]: why is this rule disabled?

**Review by GianlucaGuarini — APPROVED** [review]
well done @lukasbuenger awesome patch

File: eslint/eslint-typescript.js (hunk 1 above)
Comment (lukasbuenger) [reply eslint/eslint-typescript.js:15]: Note, that it's only disabled for plain JS Files. This rule basically enforces either ESM syntax or the TS proprietary `import foo = require('foo')` for TS files targeting Node and throws if you use regular CommonJS syntax as in `const foo = require('foo')`. That's absolutely fine for TS files but not very practical if you want to write infrastructure scripts in plain JS to download translations and such.

Comment (GianlucaGuarini) [reply eslint/eslint-typescript.js:15]: go for it thank you for your feedback

**Comment by GianlucaGuarini** [comment]
@lukasbuenger please merge it and release a new minor version with the release notes (`npm version minor && git push --tags`)

**Pull request merged**